cmd/influx_inspect/cardinality/aggregators/aggregators.go

```diff
@@ -0,0 +1,246 @@
+package aggregators
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"text/tabwriter"
+
+	"github.com/influxdata/influxdb/cmd/influx_inspect/report"
+	"github.com/influxdata/influxdb/models"
+)
+
+type rollupNodeMap map[string]RollupNode
+type rollupNodeLock interface {
```

Comment:
This is just the stdlib `Locker` interface: https://pkg.go.dev/sync#Locker

Reply:
Fixed and pushed